traffic_portal/app/src/common/modules/table/servers/TableServersController.js

```diff
@@ -240,18 +275,21 @@ var TableServersController = function(servers, $scope, $state, $uibModal, $windo
 			headerName: "TCP Port",
 			field: "tcpPort",
 			hide: true,
+			tooltip: defaultTooltip
 		},
 		{
 			headerName: "Type",
 			field: "type",
 			hide: false,
+			tooltip: defaultTooltip
```

Comment:
Instead of adding a ton of these tooltips, you should probably just add `tooltip: defaultTooltip` to `gridOptions.defaultColDef`

Reply:
oh. yeah, i was wondering if a better way

Reply:
done